PerformanceNavigationTiming for User Agent Launch/explainer.md

```diff
@@ -14,28 +15,12 @@ This document is a starting point for engaging the community and standards bodie
 * **Current version: this document**
 
 ## Introduction
-When a user agent first launches (a "cold start" scenario), it must perform many expensive initialization tasks that compete for resources on the system. As a consequence, web applications may suffer from bimodal distribution in page load performance if they attempt to load at this time as they also compete for rendering and their own initialization. This makes it difficult to detect if performance issues exist within web applications themselves or because of a user-agent condition when resources are in high contention. This is particularly a pain point for pinned PWAs or WebView apps that will often require a cold start of the user agent.
+When a user agent first launches (a "cold start" scenario), it must perform many expensive initialization tasks that compete for resources on the system. Consequently, web applications may suffer from bimodal distribution in page load performance. 
 
-## Goals
-The proposed changes in this document aim to enable web applications to:
- * have an understanding of how they were created.
- * make functional decisions when they know they are in a high resource contention scenario.
- * filter and pivot their performance telemetry on navigation type to better detect performance issues and regressions.
-
-## Non-Goals
- * Distinguishing between various implementation-specific startup processes.
- * Signaling browser launches which don't involve a full browser initialization (e.g. restoring a session from a suspended state on mobile platforms).
- * Providing other information related to browser performance conditions, outside the scenario of a cold start launch and initialization.
-
-## Use Cases
- * a web application which typically runs as an installed PWA that might get launched without a user agent already running.
- * a web application which runs as the user's homepage.
+Content they attempt to load will be in competition with other initialization work. This makes it difficult to detect if performance issues exist within web applications themselves, or because of a user-agent-generated condition of high resource contention. This is particularly a pain point for pinned PWAs (Progressive Web Apps) that will often require a cold start of the user agent, and teams we have worked with have been surprised by the tails in their dashboards. Without more information, it is challenging for developers to understand if (and when) their applications may be misbehaving or are simply being loaded in a contended period. To address this, we propose adding new information to existing Web Performance APIs. 
```

Comment:
"...and teams we have worked with have been surprised by the tails in their dashboards."

Can we say something like "and many developers are surprised at the difference between real-world dashboard metrics and what they observe in page profiling tools." ?